<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PartieOperative</class>
 <widget class="QWidget" name="PartieOperative">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PartieOperative</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>111</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Etat des capteurs</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>84</width>
      <height>112</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="checkBoxCapteur1">
       <property name="text">
        <string>Capteur 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxCapteur2">
       <property name="text">
        <string>Capteur 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxCapteur3">
       <property name="text">
        <string>Capteur 3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxCapteur4">
       <property name="text">
        <string>Capteur 4</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>401</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Barquette</string>
   </property>
   <widget class="QPushButton" name="pushButtonNouvelleBarquette">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Nouvelle Barquette</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>351</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelCodeProduit">
       <property name="text">
        <string>Code produit :</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>148</width>
         <height>22</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditCodeProduit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>351</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelNumeroEjecteur">
       <property name="text">
        <string>Numéro d'éjecteur :</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>118</width>
         <height>22</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditNumeroEjecteur"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonDebutProduction">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Début de production</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonArretProduction">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Arrêt de production</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonQuitter">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>230</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>PartieOperative</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>528</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>493</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
